iceoryx_binding_c/test/moduletests/test_wait_set.cpp

```diff
@@ -142,12 +142,28 @@ TEST_F(iox_ws_test, SizeIsZeroWhenConstructed)
 
 TEST_F(iox_ws_test, SizeIsOneWhenOneUserTriggerIsAttached)
 {
-    EXPECT_EQ(iox_ws_attach_user_trigger_event(m_sut, m_userTrigger[0U], 0U, NULL),
+    EXPECT_EQ(iox_ws_attach_user_trigger_event(m_sut, m_userTrigger[0U], 0U, userTriggerCallback),
               iox_WaitSetResult::WaitSetResult_SUCCESS);
     EXPECT_EQ(iox_ws_size(m_sut), 1U);
 }
 
 TEST_F(iox_ws_test, SizeIsOneWhenOneSubscriberStateIsAttached)
+{
+    constexpr uint64_t customId = 123U;
+    EXPECT_EQ(iox_ws_attach_subscriber_state(
+                  m_sut, &m_subscriberVector[0U], SubscriberState_HAS_DATA, customId, subscriberCallback),
+              iox_WaitSetResult::WaitSetResult_SUCCESS);
+    EXPECT_EQ(iox_ws_size(m_sut), 1U);
+}
+
+TEST_F(iox_ws_test, SizeIsOneWhenOneUserTriggerWithNullptrCallBackIsAttached)
+{
+    EXPECT_EQ(iox_ws_attach_user_trigger_event(m_sut, m_userTrigger[0U], 0U, NULL),
+              iox_WaitSetResult::WaitSetResult_SUCCESS);
+    EXPECT_EQ(iox_ws_size(m_sut), 1U);
+}
+
+TEST_F(iox_ws_test, SizeIsOneWhenOneSubscriberStateWithNullptrCallBackIsAttached)
```

Comment:
Please add a similar test for a subscriber event with a nullptr callback.

Reply:
@FerdinandSpitzschnueffler done